lib/components/molecules/Form/DragDropInput/DragDropInput.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import React from 'react';
+import type { Node } from 'react';
+import { Field } from 'formik';
+import styled from 'styled-components';
+
+import Label from '../../../atoms/Label';
+import Input from '../../../atoms/Input';
+import styles from './DragDropInput.style';
+import useOpenFileDialog from '../../../../hooks/useOpenFileDialog';
+import useDragandDrop from '../../../../hooks/useDragandDrop';
+import type { DragDropInputProps } from './types';
+
+const { openFileDialog } = useOpenFileDialog();
+
+const DDFieldInput = ({
+  id,
+  className,
+  type,
+  name,
+  label,
+  onChange,
+  onBlur,
+  labelProps,
+  fieldProps,
+  localeText,
+  ...others
+}: DragDropInputProps): Node => {
+  const fileInput = React.createRef();
+  const {
+    onDragOver,
+    onDragLeave,
+    onDrop,
+    handleKeyPress,
+    onFilesAdded,
+    getHighlightClass,
+  } = useDragandDrop();
+  const styleCls = getHighlightClass(className);
+
+  return (
+    // TODO: Add support for label position
+    <div
```

Comment:
Instead of DIV can't we use button as eventually you are giving this role as button?

Reply:
I think its drag and drop area so we should check whats the recommendation

Reply:
Changed to button and removed tabIndex, role attributes.